stories/forms/inputs/checkbox-group.story.js

```diff
@@ -75,3 +75,6 @@ storiesOf('CheckboxGroup', module)
       }}
     />
   ))
+  .add('with dropdown options', () => (
```

Comment:
The name of this story almost reads as if the options themselves have some form of dropdown capability. Could you rephrase this to be more clear?